code/processes/wdb.q

```diff
@@ -391,8 +371,8 @@ endofdaysort:{[dir;pt;tablist;writedownmode;mergelimits;hdbsettings]
 	};
 
 /-function to send reload message to rdbs/hdbs
-reloadproc:{[h;d;ptype;reloadlist]
-	.wdb.countreload:count[raze .servers.getservers[`proctype;;()!();1b;0b] each reloadlist];
+reloadproc:{[h;d;ptype]
+	.wdb.countreload:count[raze .servers.getservers[`proctype;;()!();1b;0b]each reloadorder];
```

Comment:
is this just reverting the changes originally made for the tailer?

Reply:
Yes, any changes to the wdb.q reverts it to the torq classic version